oodeel/models/__init__.py

```diff
@@ -20,3 +20,27 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+
+avail_lib = []
+try:
+    import tensorflow as tf
+
+    avail_lib.append("tensorflow")
+except ImportError:
+    pass
+
+try:
+    import torch
+
+    avail_lib.append("torch")
+except ImportError:
+    pass
+
+
+if len(avail_lib) == 2:
+    from .keras_feature_extractor import KerasFeatureExtractor
+    from .torch_feature_extractor import TorchFeatureExtractor
+elif "tensorflow" in avail_lib:
+    from .keras_feature_extractor import KerasFeatureExtractor
+elif "torch" in avail_lib:
+    from .torch_feature_extractor import TorchFeatureExtractor
```

Comment:
My comment is not a requested change, just a suggestion to make it simple.
```suggestion
try:
    import tensorflow as tf
    from .keras_feature_extractor import KerasFeatureExtractor
except ImportError:
    pass

try:
    import torch
    from .torch_feature_extractor import TorchFeatureExtractor
except ImportError:
    pass


if len(avail_lib) == 2:
    from .keras_feature_extractor import KerasFeatureExtractor
    from .torch_feature_extractor import TorchFeatureExtractor
elif "tensorflow" in avail_lib:
    from .keras_feature_extractor import KerasFeatureExtractor
elif "torch" in avail_lib:
    from .torch_feature_extractor import TorchFeatureExtractor
```

Reply:
I think you forgot to remove the last part frome line 37 to 43 in your suggestion ?

Reply:
You're right! Thanks, I've updated my comment